integration_tests/testcheck/__tests__/testcheck-fit-test.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+describe('testcheck-fit', () => {
+  check.fit('runs tests with fit exclusively', [], () => {
```

Comment:
does `check.fit` also disable other tests that are created using `it`? I'll assume the answer is yes?